eng/Version.Details.xml

```diff
@@ -386,6 +392,15 @@
       <Sha>27e584661980ee6d82c419a2a471ae505b7d122e</Sha>
       <SourceBuild RepoName="symreader" ManagedOnly="true" />
     </Dependency>
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.runtime.linux-x64" Version="9.0.0-alpha.1.23529.4">
+      <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>e5c631d28b985b773b28d0cb2346f49a92bd41dd</Sha>
+      <SourceBuild RepoName="runtime" />
+    </Dependency>
+    <Dependency Name="Microsoft.DiaSymReader" Version="2.0.0">
```

Comment:
are we going to need a darc codeflow for symreader or a dependency flow for it from runtime?

Reply:
hmmm, probably. Will know when we get a view of what comes out of the build pipeline. @Cosifne fyi.